### PR TITLE
Responsive menu

### PR DESCRIPTION
**Pull request by seqre**
Added responsive menu to main.hpp & implemented it in windyTylne.cpp

### CODE REVIEW

**Comment by seqre** [comment]
Just fyi, changing whole project for this solution and more user-friendly choice selection will be (if at all) done in another branch.

File: rooms/windyTylne.cpp [hunk 1]
```diff
@@ -71,59 +70,66 @@ bool lift_repairing_game(GameData& gameData) {  // TODO(seqre): Add minigame
...
-            << std::endl
-            << "2. " << (gameData.zaionc ? "Biegnę" : "Idę")
-            << " do windy z przodu budynku" << std::endl
-            << "3. " << (gameData.zaionc ? "Biegnę" : "Idę") << " do biblioteki"
-            << std::endl
-            << "4. " << (gameData.zaionc ? "Biegnę" : "Idę")
-            << " do garażu (i robię przyps)" << std::endl
-            << "5. Wracam " << (gameData.zaionc ? "biegiem" : "")
-            << "na korytarz" << std::endl;
+template <typename S>
```
Comment (phoe) [inline rooms/windyTylne.cpp]: This needs to be moved to `utils`.

File: rooms/windyTylne.cpp [hunk 2]
```diff
@@ -71,59 +70,66 @@ bool lift_repairing_game(GameData& gameData) {  // TODO(seqre): Add minigame
...
+  return (s + concatenate(args...));
+}
+
+int choose_room_text(GameData& gameData) {
+  return responsive_menu(
+      "Co robisz?",
+      (gameData.winda_repaired
+           ? "Korzystam z windy i zjeżdżam na parter"
+           : "Podchodzę do windy, by zobaczyć czemu nie działa"),
+      (concatenate((gameData.zaionc ? "Biegnę" : "Idę"),
```
Comment (phoe) [inline rooms/windyTylne.cpp:79]:
```cpp
(gameData.zaionc ? "Biegnę" : "Idę")
```
xD

File: rooms/windyTylne.cpp [hunk 3]
```diff
@@ -71,59 +70,66 @@ bool lift_repairing_game(GameData& gameData) {  // TODO(seqre): Add minigame
...
+                   " do windy z przodu budynku")),
+      (concatenate((gameData.zaionc ? "Biegnę" : "Idę"), " do biblioteki")),
+      (concatenate((gameData.zaionc ? "Biegnę" : "Idę"),
+                   " do garażu (i robię przyps)")),
+      (concatenate("Wracam ", (gameData.zaionc ? "biegiem" : ""),
```
Comment (phoe) [inline rooms/windyTylne.cpp:84]: Zabrakło ci tu spacji.

**Review by phoe — CHANGES_REQUESTED** [review]

File: rooms/windyTylne.cpp (hunk 2 above)
Comment (seqre) [reply rooms/windyTylne.cpp:79]: Whatever ¯\_(ツ)_/¯

File: rooms/windyTylne.cpp (hunk 1 above)
Comment (seqre) [reply rooms/windyTylne.cpp]: To `main.hpp` but ok

Comment (phoe) [reply rooms/windyTylne.cpp]: Yep - this shouldn't be a in a code file nonetheless.

File: rooms/windyTylne.cpp (hunk 3 above)
Comment (seqre) [reply rooms/windyTylne.cpp:84]: Specjalnie tak jest, ma wypisać `biegiem`, albo nic

Comment (phoe) [reply rooms/windyTylne.cpp:84]: Ale popatrz na linijkę niżej - wypisze `Wracam do ...` albo `Wracam biegiemdo ...`

**Review by phoe — APPROVED** [review]

**Pull request merged**